### PR TITLE
made the navbar sticky

### PR DESCRIPTION
**Pull request by Awesomeasma**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -19,20 +19,24 @@
...
   display: flex;
   justify-content: space-between;
   align-items: center;
   flex-wrap: wrap;
+  background-color: white;
 }
 
 .app__posts {
   padding: 20px;
+  padding-top: 71.25px;
```
Comment (narayan954) [inline src/index.css:39]: padding top isn't sufficient and try something like 5.5em

File: src/index.css [hunk 2]
```diff
@@ -381,6 +385,7 @@ div#picker {
   padding: 10px 0;
   box-shadow: 0 0 20px rgba(104, 85, 224, 0.2);
   transition: 0.4s;
+  margin-right: 10px;
```
Comment (narayan954) [inline src/index.css:388]: that's not the style for logout button, please check and correct.

**Review by narayan954 — CHANGES_REQUESTED** [review]
resolve the comments and we're good to go!

File: src/index.css (hunk 2 above)
Comment (Awesomeasma) [reply src/index.css:388]: I couldn't find the specific styles for logout button.

Comment (narayan954) [reply src/index.css:388]: then you may add a class to that button and update it in the index.css

**Comment by narayan954** [comment]
> Is this ok ? change-1 change-2

yes it should be

**Comment by Awesomeasma** [comment]
I've pushed the changes, I see that some checks have failed.. Did I do a mistake ?

**Comment by narayan954** [comment]
> I've pushed the changes, I see that some checks have failed.. Did I do a mistake ?

It's just a warning that custom components shouldn't be supplied classname as a property, as it reduces readiblity and some other isues. But I think I'll handle that by providing that style as inline style.

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Comment by Awesomeasma** [comment]
Thank you for your time and patience :)

**Comment by narayan954** [comment]
> Thank you for your time and patience : )

Our learnings are all that matters. Thanks for your contribution! :)

**Pull request merged**